func_tests/test_orders.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+import sqlalchemy
+from sqlalchemy.orm import Session
+from src.homework.db.models import Order, User
+from src.homework.db.crud import calculate_new_points
+from func_tests.test_user import insert_user  # noqa: F401
+from func_tests.general_fixtures import (  # noqa: F401
+    create_client,
+    create_master_app,
+    create_read_app,
+    create_modify_app,
+    create_db,
+    insert_app,
+)
+
+
+@pytest.fixture(name="inserted_order")
+def insert_order(eng, inserted_user):
+    with Session(eng) as session:
+        new_order = Order(
+            user_id=inserted_user, total_cost=100, completed_at=1630000000
+        )
+        session.add(new_order)
+        session.commit()
+        return new_order.order_id
+
+
+@pytest.mark.parametrize("req_type", ["post", "delete"])
+@pytest.mark.parametrize("created_app", ["read_app", "master_app"])
+def test_invalid_token(created_app, client, request, req_type):
+    created_app = request.getfixturevalue(created_app)
+    response = client.request(
+        req_type,
+        "/orders",
+        headers={"token": created_app[1]},
+        json={"user_id": 0, "order_id": 0, "total_cost": 0, "completed_at": 0},
+    )
+    assert response.status_code == 403
+
+
+@pytest.mark.parametrize(
+    "order_id, total_cost, completed_at",
+    [(1, 300, 1630000000), (2, 500, 1630000100), (3, 1000, 1630001000)],
+)
+def test_order_creation(

```

Comment:
Despite test is big, I understand why you decided to put it in one function. Now it is fine for us, but might be a refactor candidate in the future